src/Api/Library/Shared/Communicate/Communicate.php

```diff
@@ -47,47 +46,6 @@ public static function communicateToUsers(
         return $messageId;
     }
 
-    /**
-     * @param UserModel $user
-     * @param ProjectSettingsModel $project
-     * @param string $subject
-     * @param string $emailTemplate
-     * @param string $htmlEmailTemplate
-     * @param DeliveryInterface $delivery
-     */
-    public static function communicateToUser(
```

Comment:
We still need this function as this is how the system sends emails.  The function should be left alone except for checks about the COMMUNICATE_VIA property.

Reply:
are you certain?  I don't think I could find the traceback for that, the emails related to registering and such are through different methods.

Reply:
🤦 🥚 I stand corrected!  The communicateToUser() isn't used anymore as far as I can see.  The other method, `sendTemplateEmail()`, wraps `CommunicateHelper::deliverEmail()` which does the actual sending.  So this is all good.  Thanks for your patience as I get my head around all this dead wood code.  Glad we are slimming things down.